### ISSUE TITLE
Exception in inputworker: undefined method `each' for nil:NilClass

### ISSUE THREAD

**Issue by marvinwankersteen** [logstash-plugins/logstash-codec-netflow#108]
**- Version:**
docker.elastic.co/logstash/logstash:5.6.4

**- Operating System:**
Debian 9

**- Config File:**
```
input {
  udp {
    port => 9995
    codec => netflow {
      versions => [5, 9]
    }
    type => netflow
    tags => "port_9995"
  }
}



output {
  redis {
    host => "redis-asa"
    port => "6379"
    data_type => "list"
    key => "logstash"
  }
  stdout {
    codec => "rubydebug"
  }
}
```

**- Steps to Reproduce:**
I don't no how to reproduce the error. We have running a logstash container on two different servers. Each server receives netflows from a Cisco Asa and the error appears every ~3 days on both logstash-containers but not at the same time. The servers are self-sufficient.

```
[2017-11-21T23:58:55,393][WARN ][logstash.codecs.netflow  ] Invalid netflow packet received (End of file reached)
[2017-11-21T23:58:55,453][ERROR][logstash.inputs.udp      ] Exception in inputworker {"exception"=>#<NoMethodError: undefined method `each' for nil:NilClass>, "backtrace"=>["/usr/share/logstash/vendor/bundle/jruby/1.9/gems/logstash-codec-netflow-3.7.0/lib/logstash/codecs/netflow.rb:117:in `decode'", "/usr/share/logstash/vendor/bundle/jruby/1.9/gems/bindata-2.4.1/lib/bindata/array.rb:208:in `each'", "org/jruby/RubyArray.java:1613:in `each'", "/usr/share/logstash/vendor/bundle/jruby/1.9/gems/bindata-2.4.1/lib/bindata/array.rb:208:in `each'", "/usr/share/logstash/vendor/bundle/jruby/1.9/gems/logstash-codec-netflow-3.7.0/lib/logstash/codecs/netflow.rb:113:in `decode'", "/usr/share/logstash/vendor/bundle/jruby/1.9/gems/logstash-input-udp-3.1.2/lib/logstash/inputs/udp.rb:118:in `inputworker'", "/usr/share/logstash/vendor/bundle/jruby/1.9/gems/logstash-input-udp-3.1.2/lib/logstash/inputs/udp.rb:89:in `udp_listener'"]}
```

Is the backtrace from net netflow-plugin or the udp-input?

**Comment by jorritfolmer** [comment]
Seems related to #90

**Comment by jorritfolmer** [comment]
Thanks for reporting this issue.
Fixed in v3.8.3 (just released)
